src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/StreamHelper.cs

```diff
@@ -309,6 +307,13 @@ internal static void WriteToStream(Stream input, Stream output, PSCmdlet cmdlet,
             catch (OperationCanceledException)
             {
             }
+            catch (AggregateException)
+            {
+                ae.Handle((x) =>
+                {
+                    return true;  // Let all wrapped exceptions throw and be handled in calling layers
```

Comment:
From last @SteveL-MSFT comment I understand we postpone this 'right' behavior until 7.4.

Reply:
yes, that's right. 
Following has same behavior to `catch(AggregateException)`. Both will suppress AggregateException. 
Just added @PaulHigin 's suggestion to follow the pattern.
```
                ae.Handle((x) =>
                {
                    return true;
                }
```

Reply:
let me check. I think I didn't see Steve's 2nd comment today.